public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -543,7 +544,8 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
     if (interval !== 0 && range / interval > 11000) {
       interval = Math.ceil(range / 11000);
     }
-    return Math.max(interval, 1000);
+    // The min interval is set to 100ms
+    return Math.max(interval, 100);
```

Comment:
What could go wrong if we set the min to 1ms ?

Reply:
@davkal @torkelo @kaydelaney can you think of any problems that could happen if we set min to 1ms? I can't think of anything.

Reply:
@cyriltovena I have found  [this issue that Loki used to have problem with floats in the step](https://github.com/grafana/loki/issues/1305). I wasn't able to reproduce it, but I don't see any linked PR that would be fixing this on Loki side. Could it still be a case?
![image](https://user-images.githubusercontent.com/30407135/104164422-2e607400-53f8-11eb-9c32-2433cec2b2f1.png)
![image](https://user-images.githubusercontent.com/30407135/104164535-55b74100-53f8-11eb-8abc-e5794a14563d.png)

Reply:
The step is not used for log queries may be it won't work for metric queries ?

Reply:
🙈 true! But I have checked and it works for metric queries as well.
![image](https://user-images.githubusercontent.com/30407135/104378806-dbd2a500-5528-11eb-9f81-b7431b8fa617.png)

![image](https://user-images.githubusercontent.com/30407135/104379014-1fc5aa00-5529-11eb-9597-062dde25b2cb.png)

![image](https://user-images.githubusercontent.com/30407135/104379227-77641580-5529-11eb-84ba-d561458b82e5.png)

Reply:
Sounds good to me.